### PR TITLE
Scale default backend

### LINKED ISSUE #40: Scale ingress-nginx default backends
Based on number of workers.

### PR DESCRIPTION
**Pull request by jnummelin**
One per two workers might be bit overkill?

fixes #40

### CODE REVIEW

**Comment by jakolehm** [comment]
> One per two workers might be bit overkill?

Probably yes, maybe 1 per 10 or so workers (with minimum of 2)?

File: lib/pharos/addons/ingress_nginx.rb [hunk 1]
```diff
@@ -34,11 +34,21 @@ def image_name
         end
       end
 
+      # ~One replica per 10 workers, min 2
+      # @return [Integer]
+      def replicas
```
Comment (jakolehm) [inline lib/pharos/addons/ingress_nginx.rb]: Naming is a bit misleading.. this is default backend replicas, right?

Comment (jnummelin) [reply lib/pharos/addons/ingress_nginx.rb]: yes, I'll rename it

Comment (jnummelin) [reply lib/pharos/addons/ingress_nginx.rb]: fixed

**Pull request merged**